query-data/metrics/query-metrics.mdx

```diff
@@ -0,0 +1,517 @@
+---
+title: Query metrics
+description: This page explains how to query OpenTelemetry metrics.
+---
+
+In Axiom, you can use two query languages:
+- Axiom Processing Language (APL) lets you query datasets with events, OTel logs, and OTel traces. For more information, see [Introduction to APL](/apl/introduction).
+- AxQL lets you query OTel metrics. This page explains how to use AxQL.
+
+AxQL is a metric-focused query language that combines the simplicity of APL with the expressive power of PromQL. It enables effective querying, transformation, and aggregation of metric data, supporting diverse observability use cases. It addresses the shortcomings of existing metric query languages, which are either overly complex or too rigidly tied to specific storage formats.
+
+AxQL is a superset of PromQL. Existing PromQL expressions can be translated to AxQL for quick onboarding and greater flexibility. For more information, see [Migrate metrics to Axiom](/query-data/metrics/migrate-metrics).
+
+<Info>
+Support for OTel metrics is currently in public preview. For more information, see [Feature states](/platform-overview/roadmap#feature-states).
+</Info>
+
+## Limitations
+
+The current implementation of AxQL comes with the following limitations:
+
+- You can only query single metrics.
+- You can only use bucketing for histograms.
```

Comment:
This is tricky to phrase because 'histograms' are both an operation we have and a data type from hotel. 

`You can only use bucketing for histograms` reads like you can only use it on the histogram datatype to me. Bucketing has two similar but very distinctive operations:

1) Create a new histogram from a bucket of (plain, non-histogram) data
2) interpolate values from histogram data

Both are bucketing operations as they cover both time and tags, and both are histogram-related but in massively different ways